user-manual/transfer/dataverse.rst

```diff
@@ -10,20 +10,24 @@ Overview
 --------
 
 `Dataverse`_ is an open source repository for research data. Archivematica can
-be configured to use a Dataverse Repository as a Transfer Source Location. 
-Dataverse Transfer Source Locations can be configured to display all available 
+be configured to use a Dataverse Repository as a Transfer Source Location.
+Dataverse Transfer Source Locations can be configured to display all available
 datasets or a subset of them.
 
-Datasets are retrieved directly using the Dataverse API and processed like 
-standard transfers, with some additional processing as described below.
+Datasets are retrieved directly using the Dataverse API and processed using the
+Dataverse :ref:`transfer type <transfer-types>`, which enables some additional
+processing steps as described below.
 
-Dataverse integration is supported by Archivematica version 1.8 and above, and
-has been developed and tested using Dataverse version 4.8.6
+Datasets can also be exported from Dataverse and placed in a transfer source
+location that is visible to Archivematica.
```

Comment:
I think we may need to nuance this a little @sallain, Or perhaps leave it undocumented for now. I'm not sure. 

It takes some wrangling at the moment like needing to include an `agents.json` file, and extract Dataverse bundles into their component parts if bundles are used. Basically everything the Storage Service does but manually :frowning_face:

Reply:
Okay, I will remove!